src/FitterBase.cc

```diff
@@ -299,5 +305,18 @@ void FitterBase::saveChi2Contributions(std::string const &baseName) const {
     saveChi2RefContributions(refTuple);
 }
 
+float FitterBase::_lineSearch(Eigen::VectorXd const &delta) {
```

Comment:
Are you sure this should be `float`? Especially since you are only assigning it to `scale` (which is `double`) above.

Reply:
I have no idea why I mad that a float.